### PR TITLE
Added endpoints to workforce multiplier list

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?**Added endpoints to the workforce multiplier list 
- **Is this PR related to a Monolith release?**Yes, Release 2021.10.0 

### Resolves:

* [OKTA-433442](https://oktainc.atlassian.net/browse/OKTA-433442)

### CODE REVIEW

**Review by brentschaus-okta — APPROVED** [review]
LGTM

**Review by joec-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 1]
```diff
@@ -121,6 +121,10 @@ Workforce orgs that are created after January 7, 2021 have increased default rat
 * `/login/token/redirect`
 * `/login/step-up/redirect`
 * `/api/${apiVersion}/radius`
+* `/.well-known/webfinger`
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:124]: should this whole list be alphabetized?

Comment (joec-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:124]: if we can lets alphabetize A-Z

**Pull request merged**